File: app/src/main/java/com/work/workhubpro/models/Message.kt
package com.work.workhubpro.models

import java.time.LocalDateTime
import java.time.LocalTime
import java.util.Date


    data class Message(
        val msg: String,
        val sender: String,
        val receiver: String,
        var CreatedAt: LocalTime?=null,
        val ID: Int?= null,
    )

File: Backend/models/Message.go
package models

import (
	"time"

	"gorm.io/gorm"
)

type Message struct {
	gorm.Model
	Message   string    `json:"msg"`
	Sender    string    `json:"sender"`
	Receiver  string    `json:"receiver"`
	CreatedAt time.Time `json:"CreatedAt"`
}
